src/show/pages/public/Standings/index.jsx

```diff
@@ -1,161 +1,35 @@
-    name: 'Mercedes',
-    points: 157,
-    place: 1,
-    logo_url: 'http://assets.stickpng.com/images/580b585b2edbce24c47b2c52.png'
-  },
-
-  {
-    id: '3',
-    name: 'Red Bull',
-    points: 157,
-    place: 1,
-    logo_url: 'http://assets.stickpng.com/images/580b585b2edbce24c47b2c52.png'
-  },
-
-  {
-    id: '4',
-    name: 'McLaren',
-    points: 157,
-    place: 1,
-    logo_url: 'http://assets.stickpng.com/images/580b585b2edbce24c47b2c52.png'
-  }
-];
-
-const drivers = [
-  {
-    id: '1',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '2',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '3',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '4',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '5',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '6',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '7',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '8',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '9',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
+const Standings = () => {
+  const [constructor, setConstructor] = useState([]);
+  const [driver, setDriver] = useState([]);
```

Comment:
Singular for constructor and driver, yet you have an array of drivers and constructors.